code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+		icon_state = "tat_campbell"
+		body_parts= list("l_leg")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_rightleg
+		name = "Tattoo (Campbell, Right Leg)"
+		icon_state = "tat_campbell"
+		body_parts= list("r_leg")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_forrest_left
+		name = "Tattoo (Forrest, Left Eye)"
+		icon_state = "tat_forrest_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_forrest_right
+		name = "Tattoo (Forrest, Right Eye)"
+		icon_state = "tat_forrest_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hive
+		name = "Tattoo (Hive, Back)"
+		icon_state = "tat_hive"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_heart
+		name = "Tattoo (Heart, Chest)"
+		icon_state = "tat_heart"
+		body_parts = list("chest")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hunter_left
+		name = "Tattoo (Hunter, Left Eye)"
+		icon_state = "tat_hunter_l"
+		body_parts = list("head")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hunter_right
+		name = "Tattoo (Hunter, Right Eye)"
+		icon_state = "tat_hunter_r"
+		body_parts = list("head")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_jaeger_left
+		name = "Tattoo (Jaeger, Left Eye)"
+		icon_state = "tat_jaeger_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_jaeger_right
+		name = "Tattoo (Jaeger, Right Eye)"
+		icon_state = "tat_jaeger_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_joosten_left
+		name = "Tattoo (Joosten, Left Eye)"
+		icon_state = "tat_joosten_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_joosten_right
+		name = "Tattoo (Joosten, Right Eye)"
+		icon_state = "tat_joosten_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_miller_left
+		name = "Tattoo (Miller, Left Eye)"
+		icon_state = "tat_miller_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_miller_right
+		name = "Tattoo (Miller, Right Eye)"
+		icon_state = "tat_miller_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_montana_left
```

Comment:
Flat out not fitting for the roleplay enviroment, full face coverings are a no go

Reply:
Can you actually explain why.

Reply:
Unprofessional, does not fit the theme of Nanotrasen and the type of people they hire. It makes little sense as to why you would work in a research station with a flat color tattoo as part of your face